server/routes/support/handlers/auditDetails.ts

```diff
@@ -0,0 +1,25 @@
+import { Request, Response } from 'express'
+import { subYears } from 'date-fns'
+import LicenceService from '../../../services/licenceService'
+
+export default class AuditDetailsRoutes {
+  constructor(private readonly licenceService: LicenceService) {}
+
+  GET = async (req: Request, res: Response): Promise<void> => {
+    const { user } = res.locals
+    const { licenceId, auditEventId } = req.params
+    const audit = await this.licenceService.getAuditEvents(
+      parseInt(licenceId, 10),
+      null,
+      subYears(new Date(), 2),
```

Comment:
mmh.. this limit is a bit odd but guess its copied over from the other page - it'd be good to remove it from both if possible - this will never return THAT many events as it's limited to a specific licence id.

 Would you be able to fix in a follow up?